### DISCUSSION TITLE
Adobe AIR support

### DISCUSSION THREAD

**Discussion by hydroper** [erg-lang/erg#226] [Ideas]
I liked this language syntax. I'm looking for a language to replace ActionScript, which is currently the main language for Adobe AIR apps. Adobe AIR is now maintained by Harman:

- [www.airsdk.harman.com](https://airsdk.harman.com)
- [API](https://airsdk.dev/reference)
- [GitHub Support](https://github.com/airsdk/Adobe-Runtime-Support/discussions)

I've not studied Erg implementation yet... Is it possible to translate Erg into a format that can easily translate to ActionScript bytecode or SWF?

**Comment by hydroper** [comment]
Oh, I just read `erg` compiles to `.pyc`. Interesting... But as I understand, `.pyc` loses typing, so... it'd be better to target ActionScript bytecode directly.

**Comment by mtshiba** [comment]
Erg is being developed as a language that can use Python assets.
I am not familiar with ActionScript, so adding that to the compile target is difficult.
